test/admin/collections/Posts.ts

```diff
@@ -103,16 +103,49 @@ export const Posts: CollectionConfig = {
         },
       ],
     },
+    {
```

Comment:
Did you test with blocks also?

Reply:
Added a `block` into the config and adding another expect into the e2e test for the block as well.